DMX.Core.Api/Brokers/DateTimes/IDateTimeBroker.cs

```diff
@@ -0,0 +1,13 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+
+namespace DMX.Core.Api.Brokers.DateTimes
+{
+    public interface IDateTimeBroker
+    {
+        public DateTimeOffset GetCurrentDateTime(); 
+    }
+}

```

Comment:
Register the broker - that's why acceptance tests were failing.
Do the same for Portal

Reply:
It's registered now, but it looks like the build is still failing for the same reason.